reference/configuration/swiftmailer.rst

```diff
@@ -37,6 +41,15 @@ Configuration
 * `disable_delivery`_
 * `logging`_
 
+url
+~~~
+
+**type**: ``string``
+
+The URL to configure swift mailer.
```

Comment:
"The URL" doesn't look precise enough to me. What about "The URL describing the mail server to connect with"?